packages/flutter_tools/lib/src/base/io.dart

```diff
@@ -92,12 +93,34 @@ ExitFunction _exitFunction = _defaultExitFunction;
 
 /// Exits the process.
 ///
+/// Throws [StateError] if assertions are enabled and the dart:io exit
+/// is still active when called. This may indicate exit was called in
+/// a test without being configured correctly. This behavior can be
+/// removed by setting the `FLUTTER_TEST` environment
```

Comment:
Looks like the comment needs to be updated.

Reply:
Done